lib/find.js

```diff
@@ -3,36 +3,49 @@
 var requestP = require('request-promise');
 var utils = require('./utils');
 
-var url = process.env.EDUROAM != undefined ? 'http://delfi1.vvo-online.de:8080/delfi3/XML_STOPFINDER_REQUEST' : 'http://efa.vvo-online.de:8080/dvb/XML_STOPFINDER_REQUEST';
+function pointFinder(name) {
 
+    // remove whitespace, vvo-online.de handles all kinds of requests pretty nicely
+    var stopName = utils.stripSpaces(name);
 
-var find = function find(searchString, callback) {
-    searchString = utils.stripSpaces(searchString);
-
-    // for coordinates see this as well: http://data.linz.gv.at/katalog/linz_ag/linz_ag_linien/fahrplan/LINZ_AG_LINIEN_Schnittstelle_EFA_Koordinaten.pdf
     var options = {
-        url: url,
-        qs: {
-            locationServerActive: 1,
-            outputFormat: 'JSON',
-            type_sf: 'stop',
-            name_sf: searchString,
-            place_sf: 'Dresden',
-            coordOutputFormat: 'WGS84[DD.ddddd]',
-            coordOutputFormatTail: 9
-        }
-    };
+      url: 'https://webapi.vvo-online.de/tr/pointfinder?format=json',
+      qs: {
+        limit: 0,
+        query: stopName,
+        stopsOnly: true,
+        dvb: true,
+        assignedStops: true
+      }
+    }
 
     return requestP(options)
         .then(JSON.parse)
         .then(function (data) {
-            if (!data || !data.stopFinder || !data.stopFinder.points) {
-                return [];
-            }
-            var points = data.stopFinder.points;
-            if (!Array.isArray(points)) points = [points.point];
-            return points.map(utils.convertStop);
-        }).nodeify(callback);
+
+        // check status
+        if (data.Status &&
+            data.Status.Code === 'Ok' &&
+            data.Points) {
+
+            return data.Points.filter(function (p) {
+                p = p.split('|');
+                return p[2].length === 0 || p[2] === 'Dresden';
+            }).map(function (p) {
+                p = p.split('|');
```

Comment:
AFAICS this would require to add something like another `map()` use outside or iterating over the array itself, splitting the '|' accordingly. But if you have another way of doing this, feel free to apply the change or please suggest how exactly you feel that should be performed.